packages/roosterjs-content-model-types/lib/parameter/ImageEditor.ts

```diff
@@ -20,7 +20,12 @@ export type ImageEditOperation =
     /**
      * Flip an image
      */
-    | 'flip';
+    | 'flip'
+
+    /**
+     * Resize and rotate an image
+     */
+    | 'resizeAndRotate';
```

Comment:
Curious, if we want to start resize and rotate, can we let the function accepts an array and pass in ['resize', 'rotate']? So that no need to have a separate operation 'reiszeAndRotate'